### DISCUSSION TITLE
How to change opacity on .card.image-full:before  when using 'card' component with the "image-full" modifier

### DISCUSSION THREAD

**Discussion by kevinsoto832** [saadeghi/daisyui#1053] [Q&A]
Currently, when I inspect on Google Console, the I can see that the image-full modifier automatically applies an opacity of .75 on the 'before' content.

What's the most efficient way that I target this specific css class and change the opacity value to whatever I want? 

Or is there a way to ADD my own custom modifier? 
<img width="610" alt="Screen Shot 2022-08-24 at 1 54 21 PM" src="https://user-images.githubusercontent.com/43228622/186501465-33ba5841-97c7-4df7-8997-f09ab5ec2eeb.png">

couldn't find a straight forward answer on the docs.

**Comment by nenorrell** [comment]
@kevinsoto832 I know this is late, but putting this here for future use. You can override this by doing:

```typescript
<div className="card image-full [&::before]:!opacity-100 [&::before]:!bg-transparent">
...
</div>
```
